pkg/kubelet/kubelet.go

```diff
@@ -1103,6 +1123,9 @@ type Kubelet struct {
 	// dockerLegacyService contains some legacy methods for backward compatibility.
 	// It should be set only when docker is using non json-file logging driver.
 	dockerLegacyService dockershim.DockerLegacyService
+
+	// CPU Manager
+	cpuManager cpumanager.Manager
```

Comment:
I'd prefer not exposing CPU manager to the kubelet. Instead the container manager should be the interface exposed. Kubelet at a high level should deal with Volumes (Storage), Networking (CNI), Resources (Container Manager), Runtime (CRI), Metrics (cAdvisor) APIs only.

Reply:
This is done in the sub-pr (kubelet wiring)